### PR TITLE
CPLAT-7561 Add missing DomProps and SvgProps fields

### PR DESCRIPTION
**Pull request by sydneyjodon-wk**
## Motivation
  
Many React DOM and SVG attributes are missing from over_react DomProps and SvgProps and need to be added.
## Changes
  
Add missing DomProps and SvgProps field that are listed in the following:
- https://reactjs.org/docs/dom-elements.html#differences-in-attributes
- https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L2210
## Review
_[See CONTRIBUTING.md][contributing-review-types] for more details on review types (+1 / QA +1 / +10) and code review process._

  

Please review: 
@kealjones-wk @aaronlademann-wf @greglittlefield-wf @joebingham-wk 
### QA Checklist
- [ ] Tests were updated and provide good coverage of the changeset and other affected code
- [ ] Manual testing was performed if needed
    - [ ] Steps from PR author: 
        
      - [ ] CI passes
    - [ ] Anything falling under manual testing criteria [outlined in CONTRIBUTING.md][contributing-manual-testing]

## Merge Checklist
While we perform many automated checks before auto-merging, some manual checks are needed:
- [ ] A Client Platform member has reviewed these changes
- [ ] There are no unaddressed comments _- this check can be automated if reviewers use the "Request Changes" feature_
- [ ] _For release PRs -_ Version metadata in Rosie comment is correct

[contributing-review-types]: https://github.com/Workiva/over_react/blob/master/CONTRIBUTING.md#review-types
[contributing-manual-testing]: https://github.com/Workiva/over_react/blob/master/CONTRIBUTING.md#manual-testing-criteria

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: lib/src/component/prop_mixins.dart [hunk 1]
```diff
@@ -72,19 +72,26 @@ abstract class _$DomPropsMixin {
...
 
-  bool allowFullScreen, async, autoPlay, checked, controls, defer, disabled, formNoValidate, hidden, loop, multiple,
-    muted, noValidate, open, readOnly, required, seamless, selected;
+  @Accessor(key: 'default')
+  bool htmlDefault;
+
+  @Accessor(key: 'is')
+  String htmlIs;
+
+  bool allowFullScreen, async, autoPlay, checked, controls, defer, disabled, formNoValidate, headers, hidden,
```
Comment (joebingham-wk) [inline lib/src/component/prop_mixins.dart]: In this group there's headers, inputMode, integrity, keyParams, keyType, and kind which were labeled as `String` in our spreadsheet / React. Was there a reason that they ended up here?

File: lib/src/component/prop_mixins.dart [hunk 2]
```diff
@@ -72,19 +72,26 @@ abstract class _$DomPropsMixin {
...
-  String className, title, id;
+  String challenge, cite, className, controlsList, formAction, formEncType, formMethod, formTarget, nonce, summary, title, id, wrap;
 
-  dynamic accept, acceptCharset, accessKey, action, allowTransparency, alt, autoComplete, cellPadding, cellSpacing,
-    charSet, classID, colSpan, content, contentEditable, contextMenu, coords, crossOrigin, data, dateTime,
-    dir, download, draggable, encType, form, frameBorder, height, href, hrefLang, htmlFor, httpEquiv, icon, label,
-    lang, list, manifest, max, maxLength, media, mediaGroup, method, min, name, pattern, placeholder,
-    poster, preload, radioGroup, rel, role, rowSpan, sandbox, scope, scrolling, shape, sizes, spellCheck, src, srcDoc,
-    srcSet, step, tabIndex, target, type, useMap, value, width, wmode;
+  dynamic accept, acceptCharset, accessKey, action, allowTransparency, alt, autoComplete, capture, cellPadding, cellSpacing,
```
Comment (joebingham-wk) [inline lib/src/component/prop_mixins.dart]: In this section there's name, pattern, placeholder, src, srcdoc, and srcLang that are marked as `String` by TS / React but listed here as `dynamic`. Was there a reason behind that?

Comment (sydneyjodon-wk) [reply lib/src/component/prop_mixins.dart]: For both your questions, the fields you mentioned were already existing. I updated the already existing SvgProps typing, but I never did the same for the existing DomProps. I'll update those. Thank you for checking these!

Comment (sydneyjodon-wk) [reply lib/src/component/prop_mixins.dart]: Also, when I updated the typing for the existing SvgProps, I made the PR into master as well (#356). Should I do the same for these updates?

File: lib/src/component/prop_mixins.dart (hunk 1 above)
Comment (sydneyjodon-wk) [reply lib/src/component/prop_mixins.dart]: Yeah I don't know why I put them there haha. They definitely should be strings. Thank you for catching that!

File: lib/src/component/prop_mixins.dart (hunk 2 above)
Comment (sydneyjodon-wk) [reply lib/src/component/prop_mixins.dart]: PR #375 updates to these existing DomProps types in master as well.

**Comment by joebingham-wk** [comment]
@Workiva/release-management-pp

**Pull request merged**